shardingsphere-features/shardingsphere-replica-query/shardingsphere-replica-query-common/pom.xml

```diff
@@ -29,6 +29,12 @@
     <name>${project.artifactId}</name>
     
     <dependencies>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-infra-binder</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
```

Comment:
What is the useful?

Reply:
@terrymanu `Createreplicaqueryrulestatementconverter ` use `createreplicaqueryrulestatement ` I have change it to `shardingsphere-distsql-parser-statement`.